lib/telegram/bot/types/photo_size.rb

```diff
@@ -6,6 +6,16 @@ class PhotoSize < Base
         attribute :width, Integer
         attribute :height, Integer
         attribute :file_size, Integer
+
+        attr_accessor :file_id
+
+        def get_file(api)
```

Comment:
As for me it looks very dirty to pass `Api` instance as parameter.
I can't provide better solution now, just my thoughts.